Makefile

```diff
@@ -1,9 +1,8 @@
 SHELL=/bin/bash
 export PATH := ./bin:./venv/bin:$(PATH)
 
-ifndef ELASTIC_VERSION
+# Determine the version to build. Override by setting ELASTIC_VERSION env var.
 ELASTIC_VERSION := $(shell ./bin/elastic-version)
```

Comment:
Can't this be switched to `?=` as in https://github.com/elastic/elasticsearch-docker/pull/100/commits/75e8c5f0755d7a3c38546376b5223671f8d1f976 ?